### PR TITLE
Hide undesired messages during tests

### LINKED ISSUE #94: Remove ''fatal: 'master' does not have any commits" output messages during tests
Currently, when we execute tests, a lot of `fatal: your current branch 'master' does not have any commits yet` messages appear mixed with the test results, and this makes the analysis of the test run difficult.

As they really don't give any useful information, they should be hidden.

### PR DESCRIPTION
**Pull request by yeraydavidrodriguez**
Resolves #94 

This will hide all the annoying unnecessary output messages that were produced by the _syncExec_ commands used to test some repository attributes.

To do that, the stderr is redirected to the null channel, as we only need to catch exceptions in case that happen.

### CODE REVIEW

File: src/git-repo.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export class GitRepo {
       if (!existsSync(this.dir.getDirPath())) {
         throw new Error()
       }
-      execSync(`git -C ${this.getDirPath()} status`)
+      execSync(`git -C ${this.getDirPath()} status > /dev/null 2>&1`)
```
Comment (josecelano) [inline src/git-repo.ts]: hi @yeraydavidrodriguez I replaced this line:

```typescript
execSync(`git -C ${this.getDirPath()} status > /dev/null 2>&1`)
```

by:

```typescript
execSync(`git -C ${this.getDirPath()} status`, {stdio: 'ignore'})
```

File: src/git-repo.ts [hunk 2]
```diff
@@ -62,7 +62,7 @@ export class GitRepo {
       if (!existsSync(this.dir.getDirPath())) {
         throw new Error()
       }
-      execSync(`git -C ${this.dir.getDirPath()} log -n 0`)
+      execSync(`git -C ${this.dir.getDirPath()} log -n 0 > /dev/null 2>&1`)
```
Comment (josecelano) [inline src/git-repo.ts]: Same here.

**Review by josecelano — APPROVED** [review]

File: src/git-repo.ts (hunk 1 above)
Comment (yeraydavidrodriguez) [reply src/git-repo.ts]: Ok, Is very much expressive that way. Less hackery.

File: src/git-repo.ts (hunk 2 above)
Comment (yeraydavidrodriguez) [reply src/git-repo.ts]: Ok!

**Pull request merged**